src/main/java/net/sf/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -49,24 +49,24 @@
     private final FieldContentParser parser;
 
 
-    public LatexFieldFormatter() {
-        this(true);
+    public LatexFieldFormatter(JabRefPreferences prefs) {
+        this(true, prefs);
     }
 
-    private LatexFieldFormatter(boolean neverFailOnHashes) {
+    private LatexFieldFormatter(boolean neverFailOnHashes, JabRefPreferences prefs) {
```

Comment:
1. This is actual no problem but a feature. If you add a new formatter, then you just need to add a variable to the configuration object and that's it. No need to modify lots of methods to just pass the dependency.

2. That's a good point. For now think about the configuration object as a place to collect the access to the preference class. Later we might rework these classes so that they no longer dependent on the preferences. Take for example `SavePreferences`. This class encapsulates (nearly) all configuration options for saving a bib-file. Right now it depends on the preference class via `SavePreferences.fromPreferences(JabRefPreferences)` but this is the only place. Now it is easy to invert this method and change it to `SavePreferences JabRefPreferences.getSavePreferences()`.